frontend/public/components/monitoring/dashboards/index.tsx

```diff
@@ -218,38 +225,59 @@ const VariableDropdown: React.FC<VariableDropdownProps> = ({ id, name, namespace
 
   const [isError, setIsError] = React.useState(false);
 
+  // const dataSourceName = variable?.datasource?.name;
+  const dataSourceName = 'test';
```

Comment:
I think this is just for testing purposes

Reply:
Yes, this is just for testing.